src/main.rs

```diff
@@ -152,33 +175,49 @@ fn main() {
 
     for (i, day) in forecast.iter().enumerate() {
         tooltip += "\n<b>";
-        if i == 0 {
-            tooltip += &format!("{}, ", lang.today());
-        }
-        if i == 1 {
-            tooltip += &format!("{}, ", lang.tomorrow());
-        }
+        tooltip += &format!("{}, ", lang.today());
+        tooltip += &format!("{}, ", lang.tomorrow());
```

Comment:
this looks wrong, now we're printing things like "Today, Tomorrow, 2024-04-21" for every day.